mp2-v1/src/final_extraction/merge_circuit.rs

```diff
@@ -126,6 +126,8 @@ pub(crate) struct MergeCircuitInput {
 }
 
 impl MergeCircuitInput {
+    // CHORE: Remove this when relevant PR is merged
+    #[allow(dead_code)]
```

Comment:
This method as well can be removed if unused

Reply:
Alright cool will do